e2e/terraform/packer/ubuntu-jammy-amd64/setup.sh

```diff
@@ -119,10 +119,14 @@ fi
 
 echo "Configuring dnsmasq"
 
+# disable systemd stub resolver
+sudo sed -i 's|#DNSStubListener=yes|DNSStubListener=no|g' /etc/systemd/resolved.conf
```

Comment:
This is maybe nitpicky, but I'm not sure I trust the systemd project not to randomly decide that the commented-out lines should have an extra space and for that to get picked up in a seemingly innocent patch downstream. Maybe we should change this to append the `DNSStubListener=no` line?